cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -1007,6 +1007,24 @@ std::string CalculateMarcRecordHash(const MARC::Record &marc_record) {
 }
 
 
+const std::vector<std::string> UNDESIRED_ITEM_TYPES {
```

Comment:
There should be no space before the opening curly bracket.

Reply:
OK, but then i will also change this in other occurrences i see